game/scripts/npc/abilities/keeper_of_the_light_blinding_light.txt

```diff
@@ -38,7 +38,7 @@
       "01"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "miss_rate"                                       "50" //OAA
+        "miss_rate"                                       "65" //OAA
```

Comment:
Just make it 70% Miss chance. Then its a 100% blind with the talent, everyone will be getting a dispel or true strike anyway. It will be nice for bosses.

Reply:
> Just make it 70% Miss chance. Then its a 100% blind with the talent, everyone will be getting a dispel or true strike anyway. It will be nice for bosses.

That was my intention but I didn't want a spell to just have 100% miss chance so I made it 95%. If people are ok with having full miss chance than I am fine with that as well.